styles/protocol.css

```diff
@@ -32,6 +32,12 @@
   border-bottom: var(--drawer-toolbar-border-color);
 }
 
+#drawerContainer {
```

Comment:
Hm not sure why I removed it, but it seems to be working just fine with this change 😄